src/tests/GibsonSceneTest.cpp

```diff
@@ -49,10 +50,10 @@ void GibsonSceneTest::testGibsonScene() {
   CORRADE_VERIFY(semanticScene.objects().size() != 2);
   auto object = semanticScene.objects()[1];
   CORRADE_COMPARE(object->category()->name(""), "microwave");
-  CORRADE_VERIFY(
-      object->obb().center().isApprox(esp::vec3f(2.83999, 4.76085, 1.49223)));
-  CORRADE_VERIFY(
-      object->obb().sizes().isApprox(esp::vec3f(0.406775, 1.28023, 0.454744)));
+  CORRADE_VERIFY(object->obb().center() ==
```

Comment:
Try to use `CORRADE_COMPARE()` in all these cases, this has useless diagnostic output in case the comparison fails.